src/lib/types/experimental.ts

```diff
@@ -64,15 +65,17 @@ export const defaultExperimentalOptions = {
 
 export interface IExperimentalOptions {
     flags: {
-        [key: string]: boolean;
+        E?: boolean;
```

Comment:
It it supposed to be some generic or a typo?

Reply:
Discussed in: https://unleash-internal.slack.com/archives/C046LV6HH6W/p1669880769980979 - Needed to include this option since it's being passed in:
https://github.com/Unleash/unleash/blob/bc3744d56566b6a173f5473344557f0ad7f72e0b/src/lib/create-config.ts#L101-L112